package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bulbs-elements",
-  "version": "0.1.1",
+  "version": "0.2.4",
```

Comment:
Until we're ready to pull the trigger on a production deploy, let's stick to "X.Y.Z-rc#". It should be safe to roll back from X.Y.4 to X.Y.3, but if we did a bunch of release candidates in between we won't know what version to roll back to.